MetaMask/metamask-extension
# Fix layer 2 fee alignment

## Explanation

This PR fixes alignment for layer 2 gas fee value.
It also sets layer 2 gas fee decimals to be 12 instead of 6.


Fixes #17426 
## Screenshots/Screencaps



### Before
![image](https://user-images.githubusercontent.com/97883527/225254875-b70bbaa6-acab-40bc-9201-ccc120dfaa33.png)



### After
<img width="358" alt="image" src="https://user-images.githubusercontent.com/97883527/225255005-bfe5e68c-b6f5-4c35-ab29-89de27233693.png">




## Manual Testing Steps



1. Change to Optimism network
2. Go to Etherscan to interact with an ERC20 token https://optimistic.etherscan.io/token/0x94b008aa00579c1307b0ef2c499ad98a8ce58e58#writeContract
3. Click Connect MM
4. Accept MM
5. Click Approve and Add spender and amount
6. Click Write
7. Add any amount on MM
8. Click Next
9. See alignment for fees

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
